EtaLearning.API/Controllers/ClientsController.cs

```diff
@@ -1,77 +1,118 @@
-﻿using EtaLearning.API.Data;
+﻿using EtaLearning.Core.Service;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
 
 
-[ApiController]
-[Route("api/[controller]")]
-public class ClientsController : ControllerBase
+namespace EtaLearning.API.Controllers
 {
-    private readonly IClientRepository _clientRepository;
-
-    public ClientsController(IClientRepository clientRepository)
-    {
-        _clientRepository = clientRepository;
-    }
-
-    [HttpGet("GetClients")]
-    public IActionResult GetClients()
+    [ApiController]
+    [Route("api/[controller]")]
```

Comment:
You should specify specific route here. It's part of a url that user is going to open in browser in order to call a method inside this controller. You shouldn't use special characters here except for "-" instead of space if it's more then 1 word. And you should call it the same name as your controller name. In this case it's "clients".